lib/experiments/index.js

```diff
@@ -1,11 +1,22 @@
 'use strict';
 
-// eslint-disable-next-line no-unused-vars
-const symbol = require('../utilities/symbol');
+// eslint-disable-next-line no-unused
+function isExperimentEnabled(experimentName) {
+  if (process.env.EMBER_CLI_ENABLE_ALL_EXPERIMENTS) {
+    return true;
+  }
+
+  let experimentEnvironmentVariable = 'EMBER_CLI_' + experimentName;
+  if (process.env[experimentEnvironmentVariable]) {
+    return true;
+  }
+
+  return false;
+}
 
 let experiments = {
-  MODULE_UNIFICATION: symbol('module-unification'),
-  DELAYED_TRANSPILATION: symbol('delayed-transpilation'),
+  MODULE_UNIFICATION: isExperimentEnabled('MODULE_UNIFICATION'),
```

Comment:
we will likely need to update the MU usage instructions, because now canary does not use these experiments by default anymore

Reply:
Yep, I discussed this with @mixonic and @real_ate the other day. I _believe_ that @mixonic is on it...